### ISSUE TITLE
Slow ArcGIS Load for Aggregated FS

### ISSUE THREAD

**Issue by dmfenton** [koopjs/koop-provider-composite#4]
Smartmapping Symobolization in ArcGIS map doesn’t seem to be working, I think because it can’t read the values quickly enough. 

Only giving “other” option when trying to symbolize by county
https://adapters.koopernetes.com/composite/addresses/featureserver/0 
https://usnsdi.maps.arcgis.com/home/webmap/viewer.html?webmap=863877a12b7c41428d5e674baae27b4f 

![image](https://user-images.githubusercontent.com/8593748/30281347-22acb14c-96e0-11e7-9048-92d4cfac9d15.png)

**Comment by MarkTorrey** [comment]
Hey @dmfenton, quick question. Looking at getting the unique value rendering working. I've been testing with translating the fields for the outStatistic queries, which looks to be returning the correct counts. However, there is a call to classificationDef after that I'm thinking we need to translate as well? Does this sound correct, or am I headed down the wrong path?

**Comment by dmfenton** [comment]
I don't think you can do aggregations like this. I think we may need to fetch all features and let Koop handle aggregations. otherwise you have to find a way to average teh statistics across all the different feature services you're requesting data from.
